packages/core/demo/index.ts

```diff
@@ -292,6 +297,12 @@ chartTypes.forEach(type => {
 						options: Object.assign({}, type.options, {type: type.id})
 					}
 				);
+				const pieChartObject = classyCharts[type.id];
+				pieChartObject.events.addEventListener("onClick", e=> {
+					console.log("Im clicked");
+					console.log(e.detail);
```

Comment:
1. We should use more descriptive text here, e.g. `Pie slice clicked`.
2. `console.log("Im clicked", e.detail);` would be simpler
3. The name `onClick` is misleading since it could mean catching a click anywhere on the pie chart, should use `slice-onClick`